genson/src/main/java/com/owlike/genson/Wrapper.java

```diff
@@ -23,7 +23,7 @@
  */
 public abstract class Wrapper<T> implements AnnotatedElement {
   private AnnotatedElement wrappedElement;
-  protected T wrapped;
+  protected volatile T wrapped;
```

Comment:
If we make sure that initialization is always thread safe then this wouldn't need to be volatile (which does have a small overhead).

Reply:
Agree, volatile here was added so dont need lock/synchronize on every create() call of circular converter. Same could be achieved by adding volatile flag to circular converter only, than this would not affect any other converters